### PR TITLE
fix(admin-scripts): Work with decentralized proposer

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

Small bug fix.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [X]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/scripts/src/admin-proposals/addContractCreator.js [hunk 1]
```diff
@@ -158,25 +163,14 @@ async function run() {
     }
 
     // Send the proposal
-    console.group(`\n📨 Sending to governor @ ${mainnetContracts.governor.options.address}`);
+    console.group(`\n📨 Sending to proposer @ ${mainnetContracts.proposer.options.address}`);
     console.log(`- Admin proposal contains ${adminProposalTransactions.length} transactions`);
```
Comment (mrice32) [inline packages/scripts/src/admin-proposals/addContractCreator.js:167]: Should you put both of these logs inside the proposeAdminTransactions method?

**Review by mrice32 — APPROVED** [review]
Looks good!

File: packages/scripts/src/admin-proposals/addContractCreator.js (hunk 1 above)
Comment (nicholaspai) [reply packages/scripts/src/admin-proposals/addContractCreator.js:167]: done

**Comment by nicholaspai** [comment]
@mrice32 I added in `resolveProposal` logic

File: packages/scripts/src/utils/seedProposerWithUma.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const ExpandedERC20 = getContract("ExpandedERC20");
+
+async function _seedProposerWithUma(proposer, amountToSend = toWei("50000")) {
+  const uma = new web3.eth.Contract(ExpandedERC20.abi, await _getContractAddressByName("VotingToken", 1));
+  console.log(`🧧 Sending ${amountToSend} of UMA to ${proposer} from foundation wallet 🧧`);
+  const accounts = await web3.eth.getAccounts();
+  await web3.eth.sendTransaction({
+    from: accounts[0],
+    to: REQUIRED_SIGNER_ADDRESSES["foundation"],
+    value: toWei("10"),
```
Comment (chrismaree) [inline packages/scripts/src/utils/seedProposerWithUma.js:17]: why this?

**Review by chrismaree — APPROVED** [review]
this looks great1

File: packages/scripts/src/admin-proposals/utils.js [hunk 3]
```diff
@@ -247,6 +253,84 @@ const verifyGovernanceRootTunnelMessage = async (targetAddress, message, governo
...
+  const governor = new web3.eth.Contract(Governor.abi, await _getContractAddressByName("Governor", 1));
+
+  console.group(`\nResolving the latest ${proposalsToLookback} proposals`);
+  const totalProposals = Number(await governor.methods.numProposals().call());
+  const bondedProposals = await Promise.all(
+    [...Array(totalProposals).keys()].map((i) => proposer.methods.bondedProposals(i).call())
+  );
+  for (let i = bondedProposals.length - 1; i >= bondedProposals.length - 1 - proposalsToLookback; i--) {
+    const bondedProposal = bondedProposals[i];
+    if (toBN(bondedProposal.lockedBond.toString()).gt(toBN(0))) {
```
Comment (mrice32) [inline packages/scripts/src/admin-proposals/utils.js:315]: So this settles any unsettled proposal? Do you think we should only settle proposals done by whitelisted addresses (or just the caller)? Or should we just resolve any and all by default?

File: packages/scripts/src/admin-proposals/utils.js [hunk 4]
```diff
@@ -247,6 +253,84 @@ const verifyGovernanceRootTunnelMessage = async (targetAddress, message, governo
...
+      console.log(
+        `- Proposal #${i} has a locked bond of ${bondedProposal.lockedBond.toString()} UMA, proposer was @ ${
+          bondedProposal.sender
+        }`
+      );
+      try {
+        const txn = await proposer.methods.resolveProposal(i).send({ from: caller });
```
Comment (mrice32) [inline packages/scripts/src/admin-proposals/utils.js:322]: Could be faster and more gas efficient to use multicall here (we could simulate the txn before adding it to the group of txns to send with multicall). Not a must-have though, so we could add this later.

**Review by mrice32 — APPROVED** [review]
LGTM!

File: packages/scripts/src/admin-proposals/utils.js (hunk 3 above)
Comment (nicholaspai) [reply packages/scripts/src/admin-proposals/utils.js:319]: lets just resolve by caller

**Pull request merged**